AElf.Kernel/GenesisBlock.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using AElf.Kernel.Extensions;
+using AElf.Kernel.KernelAccount;
+
+namespace AElf.Kernel
+{
+    public class GenesisBlock : IBlock
+    {
+        private readonly BlockHeader _blockHeader = new BlockHeader(Hash<IBlock>.Zero);
+        private readonly BlockBody _blockBody = new BlockBody();
+        private AccountZero _accountZero;
+        private readonly WorldState _worldState;
+
+        public GenesisBlock(WorldState worldState, AccountZero accountZero)
+        {
+            _worldState = worldState;
```

Comment:
WorldState And AccountZero should in chain, not in a block.

GenesisBlock should have a transaction to deploy AccountZero

Reply:
I will try